orchagent/observer.h

```diff
@@ -16,8 +18,14 @@ enum SubjectType
     SUBJECT_TYPE_MIRROR_SESSION_CHANGE,
     SUBJECT_TYPE_INT_SESSION_CHANGE,
     SUBJECT_TYPE_PORT_CHANGE,
+    SUBJECT_TYPE_BRIDGE_PORT_CHANGE,
```

Comment:
Who is sending this update? Can you point me to code?

Reply:
@prsunny : This is been processed in isolationgrouporch.cpp:233.  The update is sent from portsOrch part of addBridgePort and removeBridgePort calls. thanks.